plugins/protocolbuffers/python/v25.0/buf.plugin.yaml

```diff
@@ -14,4 +14,4 @@ registry:
     requires_python: ">=3.8"
     deps:
       # https://pypi.org/project/protobuf/
-      - "protobuf"
+      - "protobuf~=4.25"
```

Comment:
Which version of protocolbuffers/python is comaptible with protobuf >=5?

Reply:
Never mind, this is irrelevant. I double-checked what we're doing and this makes sense.